From e1c720822608a8930b1c1766cd520ce9aa732cf7 Mon Sep 17 00:00:00 2001
From: Charles Stevens <charles.stevens@compasses.com>
Date: Mon, 19 Jul 2021 17:53:50 -0500
Subject: [PATCH 49/57] RSOM-222 - change RGB display compatible string

Call out the new display added to panel-simple driver for Beacon
kits.

Signed-off-by: Charles Stevens <charles.stevens@compasses.com>
---
 arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)
 mode change 100755 => 100644 arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi

diff --git a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
old mode 100755
new mode 100644
index 418bb3ac37e6..6192d1e2febd
--- a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
+++ b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
@@ -136,8 +136,7 @@
 	};
 
 	rgb: display-rgb {
-		/* Some random LCD with similar timings */
-		compatible = "rocktech,rk070er9427";
+		compatible = "aplus,AF50BSIAGDNG0";
 		backlight = <&backlight_dpi>;
 		enable-gpios = <&gpio1 21 GPIO_ACTIVE_HIGH>;
 		status = "disabled";
-- 
2.17.1

